code/game/machinery/pipe/construction.dm

```diff
@@ -299,6 +299,27 @@ Buildable meters
 		C.blood_volume = 0
 	return(OXYLOSS|BRUTELOSS)
 
+/obj/item/pipe/examine(mob/user)
+	. = ..()
+	. += span_notice("The pipe layer is set to [piping_layer].")
+	. += span_notice("You can change the pipe layer by Alt-Clicking the device.")
+	. += span_notice("You can rotate the device by using it in hand.")
```

Comment:
"the device"?

Reply:
dunno how else call them, since they are atmospheric devices

Reply:
"rotate it"?

Reply:
How about just "rotate it"? Or are you referring to like, accessories

Reply:
i'll go with rotate it